sale_order_product_availability_inline/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Order Product Availability Inline",
+    "summary": "Show product availability in sales order line product " "drop-down.",
+    "version": "15.0.0.0.1",
+    "development_status": "Beta",
```

Comment:
Mybe you can change status to maturity??

Reply:
Done